apis/v1alpha1/gateway_types.go

```diff
@@ -85,8 +85,8 @@ type GatewaySpec struct {
 	//
 	// Support: Core
 	//
-	// +required
 	// +kubebuilder:validation:MinItems=1
+	// +kubebuilder:validation:MaxItems=16
```

Comment:
Is there any defined spec for how 1 gateway with 2 listeners compares to 2 gateways with 1 listener each?

Reply:
I don't think we've defined that very well in the spec yet. It seems like something that would be fairly implementation specific though. Although we'd initially discussed the possibility of merging Gateways that seems to no longer be necessary with this listener abstraction.